lambda_species/lambda_function.py

```diff
@@ -0,0 +1,48 @@
+import json
+import boto3
+
+def lambda_handler(event, context):
+	# Create a connection to the DynamoDB instance of AWS
+	dynamodb = boto3.resource('dynamodb')
+	
+	# Create a connection to the specific table we want from our DynamoDB. In this case, 'CrowFacts'.
+	table = dynamodb.Table('CrowFactsTwo')
+	
+	print(table.table_name, table.table_id, table.item_count, table.key_schema)
+	
+	response = None
+	code = 200
+	
+	# Check that the incoming request has the necessary parameters. 
+    # In this case we are expecting 'CrowSpecies' and 'habitat'. 
+	# For a GET method, the incoming parameters need to be the primary/sorting keys of the database.
+	# For a POST method, the incoming parameters need to be the new data you want to add to the database
+	
+	if 'CrowSpecies' in event and 'location' in event:
```

Comment:
This doesn't match. If you want to look for `CrowSpecies` and `habitat`, then no need to check for `location`
```suggestion
	if 'CrowSpecies' in event and 'habitat' in event:
```

Reply:
LOL, you're totally right, I caught this upstream after opening the PR and thought to myself "I should fix this in a commit"